charts/fluent-bit/aws-marketplace/values.yaml

```diff
@@ -0,0 +1,161 @@
+namespace:
+  create : true
+  name: "calyptia-fluentbit"
+
+serviceAccount:
+  name: "eks-calyptia-fluentbit"
+  create: true
+  annotations:
+    eks.amazonaws.com/role-arn: "<Enter the ARN of the IAM service account>"
```

Comment:
Let's provide a fully worked example for people who would not know how to get this.

Are we saying this SA must be created first or can we just add the ARN annotation?